browser/help/topics/v3-ht-annotations.md

```diff
@@ -159,17 +180,21 @@ The gnomAD v3 Hail Table annotations are defined below:
-- **revel**:
-  - **revel_score**: dbNSFP's Revel score from 0 to 1. Variants with higher scores are predicted to be more likely to be deleterious.
-  - **has_duplicate**: Whether the variant has more than one revel_score associated with it.
-- **splice_ai**:
-  - **splice_ai**: The maximum delta score, interpreted as the probability of the variant being splice-altering.
-  - **splice_consequence**: The consequence term associated with the max delta score in 'splice_ai_max_ds'.
-  - **has_duplicate**: Whether the variant has more than one splice_ai score associated with it.
-- **primate_ai**:
-  - **primate_ai_score**: PrimateAI's deleteriousness score from 0 (less deleterious) to 1 (more deleterious).
-  - **has_duplicate**: Whether the variant has more than one primate_ai_score associated with it.
+- **in_silico_predictors**: Variant prediction annotations. Struct contains prediction scores from multiple in silico predictors for variants that are predicted to be missense, impacting protein function, evolutionarily conserved, or splice-altering. We chose scores for either MANE Select or canonical transcripts if a prediction score was available for multiple transcripts.
+  - **revel**: Rare Exome Variant Ensemble Learner (REVEL) [score](https://www.cell.com/ajhg/fulltext/S0002-9297(16)30370-6). An ensemble score for predicting the pathogenicity of missense variants (based on 13 other variant predictors).
+    - **revel_max**: Score predicting the pathogenicity of missense variants with a range of 0 to 1. Variants with higher scores are predicted to be more likely to be deleterious. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript.
+  - **primateai_3d**: A deep neural network-trained [score](https://www.science.org/doi/10.1126/science.abn8197) for classifying pathogenicity of missense variants.
+    - **primateai-3d_max**: Score ranging from 0 to 1 predicting the deleteriousness of missense variants. Higher scores indicate a variant is more likely to be deleterious. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript. 
+  - **cadd**: Combined Annotation Dependent Deletion [score](https://academic.oup.com/nar/article/47/D1/D886/5146191) used to predict deleteriousness of SNVs and indels.
+    - **phred**: CADD Phred-like scaled C-scores ranging from 1 to 99 based on the rank of each variant relative to all possible 8.6 billion substitutions in the human reference genome. Larger values indicate increased predicted deleteriousness.
+    - **raw_score**: Unscaled CADD scores indicating whether a variant is likely to be "observed" (negative values) vs "simulated" (positive values). Larger values indicate increased predicted deleteriousness.
+  - **sift**: Sorting Tolerant From Intolerant (SIFT) [score](https://www.nature.com/articles/nprot.2009.86) used to predict the impact of SNVs on protein function.
```

Comment:
looking at this again, none of the in silico scores really need to have additional nesting if they only contain a single score, right? for example, why not have REVEL_max as a top level annotation within the in silico predictors struct?

Reply:
It's because of CADD, we're still keeping two scores. I only to keep each score in its own struct.

Reply:
You want to make something like cadd_phred, cadd_raw_score, so every score can be under insico_predictors?

Reply:
no, I meant, for the scores that only have one value, why not do:
in silico struct
    - revel max
    - primateAI 3D max
    - CADD
        - raw
        - phred
    - phyloP
    - etc
?

Reply:
We could, it just the way we process each score, we put each in a struct.

Reply:
I will defer to @mike-w-wilson's opinion then (since he has been reviewing those PRs)

Reply:
Historically, I also had other items under each predictor.

Reply:
yeah, it makes sense to have an entire struct for each predictor if there were multiple fields, but it seems a little unnecessary to now nest the scores if there is only one score for most predictors. I agree CADD should still have a struct

Reply:
I will wait for Mike's opinion if we should change all of them to non-nested in the processing.

Reply:
so far, only CADD and REVEL have been run.

Reply:
I removed the struct for other predictors.